projects/igniteui-angular/src/lib/core/utils.ts

```diff
@@ -262,6 +262,13 @@ export interface CancelableEventArgs {
     cancel: boolean;
 }
 
+export interface IBaseEventArgs {
+    /**
+     * Provides reference to the context component.
+     */
+    context?: any;
```

Comment:
@MayaKirova: We discussed this and will want to use "owner" instead. For the event arguments that already have "owner" we will make a breaking change and document it.